exercises/practice/bob/.approaches/switch-on-tuple/content.md

```diff
@@ -0,0 +1,44 @@
+# `switch` on a `tuple`
+
+```csharp
+    // uses System.Linq;
+    private static bool isShout(string input) => input.Any(c => char.IsLetter(c)) && input.ToUpper() == input;
+
+    public static string Response(string statement)
+    {
+        var input = statement.TrimEnd();
+        if (input == "") return "Fine. Be that way!";
+
+        switch ((input.EndsWith('?'), isShout(input)))
+        {
+            case (true, true): return "Calm down, I know what I'm doing!";
+            case (_, true): return "Whoa, chill out!";
+            case (true, _): return "Sure.";
+            default: return "Whatever.";
+        }
+    }
+```
+
```

Comment:
I don't know enough about C# to suggest a good summary here, but I think it would be useful to have one.